react/tic-tac-toe/src/app/screens/Home/screens/Player/index.js

```diff
@@ -1,20 +1,95 @@
-import React from 'react';
+import React, { Component } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+
+import matchesActions from '../../../../../redux/Matches/actions';
 
 import styles from './styles.module.scss';
 
-function Player({
-  match: {
-    params: { name }
+class Player extends Component {
+  componentDidMount() {
+    if (!this.props.data.length) {
+      this.getData();
+    }
+  }
+
+  getData = () => {
+    this.props.getMatches();
+  };
+
+  lostMatches = () => {
+    const {
+      match: {
+        params: { name }
+      },
+      data
+    } = this.props;
+    const { won, tie, lost } = data.reduce(
+      (acum, curr) => {
+        if (curr.player_one === name || curr.player_two === name) {
+          if (curr.winner === 'tie') {
+            acum.tie = acum.tie + 1;
+          } else if (curr[curr.winner] === name) {
+            acum.won = acum.won + 1;
+          } else {
+            acum.lost = acum.lost + 1;
+          }
+        }
+        return acum;
+      },
+      { won: 0, tie: 0, lost: 0 }
+    );
+    return (
+      <>
+        <div className={styles.infoBullet}>
```

Comment:
jsx code should be defined in the render method. If you think it would become too big and unreadable, extract it to another component.